### ISSUE TITLE
dcterms created and modified timestamps incorrect

### ISSUE THREAD

**Issue by trikkegitaar** [jmcnamara/libxlsxwriter#110]
In core.c the static function _localtime_to_iso8601_date is used to convert `time_t` to ISO8601 timestamp.
The `time_t` (epoch UTC) is first converted to local time `struct tm` and then to a string, but with a Z-suffix. This is wrong. In order to use the Z-suffix, the time stamp must be UTC time, not local. 
To convert the `time_t` to a UTC time `struct tm`, the `gmtime` function could be used instead of `localtime`.

I'd suggest to change the function to:
```C
static void
_gmtime_to_iso8601_date(time_t *timer, char *str, size_t size)
{
    struct tm *tmp_gmtime;
    time_t current_time = time(NULL);    
    if (*timer)
        tmp_gmtime = gmtime(timer);
    else
        tmp_gmtime = gmtime(&current_time);
    strftime(str, size - 1, "%Y-%m-%dT%H:%M:%SZ", tmp_gmtime);
}
```

**Comment by jmcnamara** [comment]
Hi,

Thanks. I had a similar issue in the Perl and Python versions as well but forgot to fix it in the C version.

I'll look into it.

Thanks,

John

**Comment by jmcnamara** [comment]
Fixed in version 0.7.1.

Thanks for the report.
